migrations/sql/V2020.11.03.08.43__clean_up_parties.sql

```diff
@@ -0,0 +1,38 @@
+UPDATE etl_permit SET party_guid=uuid(distinct_parties.party_guid) FROM (select DISTINCT count(*), party_combo_id, MAX(party_guid::text) as party_guid FROM etl_permit GROUP BY party_combo_id) AS distinct_parties
+WHERE etl_permit.party_combo_id IS NOT NULL AND etl_permit.party_combo_id != '' and etl_permit.party_combo_id = distinct_parties.party_combo_id;
+
```

Comment:
I don't think this will run on a fresh db.

Reply:
smart